src/components/hostFamilyCard/HostFamilyCard.tsx

```diff
@@ -20,53 +20,55 @@ type Props = {
 
 export default function HostFamilyCard({ hostFamily }: Props) {
   const defaultImage = "/images/family.jpg";
+  const navigate = useNavigate();
 
   return (
-    <Card className="relative flex flex-col overflow-hidden max-w-[600px]">
-      <CardHeader className="w-full">
-        <img
-          className="w-full object-cover object-center"
-          src={hostFamily.photoUrl || defaultImage}
-          alt={hostFamily.displayName ?? "Family"}
-        />
-      </CardHeader>
+    <Card className="relative flex flex-col overflow-hidden cursor-pointer max-w-[600px]">
+      <Link to={`/family-infos/${hostFamily.uid}`} state={hostFamily}>
+        <CardHeader className="w-full">
+          <img
+            className="w-full object-cover object-center"
+            src={hostFamily.photoUrl || defaultImage}
+            alt={hostFamily.displayName ?? "Family"}
+          />
+        </CardHeader>
 
-      <CardContent className="flex-grow">
-        <CardContent>
-          <div className="flex flex-col lg:flex-row sm:justify-between items-center">
-            <div className="flex gap-2 mb-4 lg:mb-0 flex-col">
-              <ItemInfo
-                nativeLanguage={hostFamily.familyLanguage ?? "Langues ?"}
-                icon={<Flag />}
-              />
-              <ItemInfo region={hostFamily.region} icon={<MapPin />} />
-              <ItemInfo price={hostFamily.familyDailyRate ?? 0} icon={<Euro />}>
-                / jour
-              </ItemInfo>
+        <CardContent className="flex-grow">
+          <CardContent>
+            <div className="flex flex-col lg:flex-row sm:justify-between items-center">
+              <div className="flex gap-2 mb-4 lg:mb-0 flex-col">
+                <ItemInfo
+                  nativeLanguage={hostFamily.familyLanguage ?? "Langues ?"}
+                  icon={<Flag />}
+                />
+                <ItemInfo region={hostFamily.region} icon={<MapPin />} />
+                <ItemInfo
+                  price={hostFamily.familyDailyRate ?? 0}
+                  icon={<Euro />}
+                >
+                  / jour
+                </ItemInfo>
+              </div>
+              <Button
+                className="w-full lg:w-fit"
+                onClick={() => navigate("/not-sale")}
```

Comment:
Normalement Aurelia a supprimer /not-sale puisque le bouton affiche soit 
- Plus d'infos et => la page de la famille
- Soit Reserver (page de la famille) et => fake stripe